pkg/apis/apps/v1beta2/defaults_test.go

```diff
@@ -173,7 +176,9 @@ func TestSetDefaultDaemonSetSpec(t *testing.T) {
 }
 
 func TestSetDefaultStatefulSet(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.MaxUnavailableStatefulSet, true)()
```

Comment:
Same as Aldo's comments earlier. This needs to be run with the featuregate enabled and disabled.